include/os_io.h

```diff
@@ -1,4 +1,5 @@
 #pragma once
+#include "app_config.h"

```

Comment:
Could be below toolchain includes and just above `#include "bolos_target.h"`?

Reply:
done